### PR TITLE
Fix cost step description

### PR DESCRIPTION
**Pull request by PanSpagetka**
Screenshot from 2020-10-14 14-05-21

### CODE REVIEW

File: src/smart-components/role/add-role-new/schema.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export default (container) => ({
       fields: [
         {
           title: 'Create role',
+          description: 'BLABKABA',
```
Comment (rvsia) [inline src/smart-components/role/add-role-new/schema.js:18]: Is this some secret acronym? :smiley:

File: src/smart-components/role/add-role-new/schema.js [hunk 2]
```diff
@@ -154,11 +155,15 @@ export default (container) => ({
           title: 'Define Cost Management resources',
           nextStep: 'review',
           fields: [
+            {
+              component: 'plain-text',
+              name: 'text-description',
+              // eslint-disable-next-line quotes
+              label: "Specify where you'd like to apply each cost permission selected int the previous step, suig the dropdown below.",
```
Comment (rvsia) [inline src/smart-components/role/add-role-new/schema.js:162]: typo `int` > `in`? 

`suig`?

**Review by rvsia — APPROVED** [review]

**Comment by mmenestr** [comment]
Just seeing this, there's a typo in the description - should be "using" not "suig"

**Comment by rvsia** [comment]
@mmenestr It's fixed in the code: https://github.com/RedHatInsights/insights-rbac-ui/pull/535/files#diff-55017a5ada9c10ef801edc3d5dca87ab8d2b153e3adce91ed8e1e4b529c4d20dR161

**Pull request merged**